README.rst

```diff
@@ -113,6 +113,20 @@ Telegram API support
 
 All types and methods of the Telegram Bot API **5.7** are supported.
 
+===========
+Concurrency
+===========
```

Comment:
imo this section should come after learning by example at the very least, if not the wiki directly

Reply:
Would you be okay with fine tuning that later on in a doc-fixes PR? Specifically, I'm still thinking of unifying the readme & the startpage of RTD - see first bullet point of #2633

Reply:
alright